DomainContent/AvatarStore/ShopZone/shopZone.js

```diff
@@ -10,7 +10,14 @@
 
 (function () {
     var shared = Script.require('../attachmentZoneShared.js');
-    this.leaveEntity = function (entityID) {
+
+    this.enterEntity = function() {
+        if (!HMD.active) {
+            Messages.sendLocalMessage('com.highfidelity.wear.tutorialChannel', 'storeEnter');
+        }
+    };
+
+    this.leaveEntity = function(entityID) {
```

Comment:
Will this zone encompass the entire island?

Reply:
no, this is only for the shop